src/core/expr/fexpr_fillna.cc

```diff
@@ -46,9 +53,16 @@ class FExpr_FillNA : public FExpr_Func {
       std::string out = "fillna";
       out += '(';
       out += arg_->repr();
+      bool hasValue = (value_->get_expr_kind() != Kind::None);
+      if (hasValue) {
+      out +=", value=";
+      out += value_->repr();
+      out += ')';
+      } else {
       out += ", reverse=";
       out += reverse_? "True" : "False";
       out += ')';
+      }
```

Comment:
The logic here seems to be wrong. When the `value` is provided, the `reverse` parameter will never be included in `out`. You need to move the `reverse` handling out of the `else` block. 

I would actually propose to remove the `if` condition fully, and add the `value` to `out` in all the cases, just add the `value` as `None`, if it is not provided.

Reply:
Btw, we should throw an error, in the case both `value` and `reverse` parameters are provided. That's because if a `value` is specified, the `reverse` parameter has no effect.